src/components/impact/styledImpact.js

```diff
@@ -1,28 +1,92 @@
 import styled from "styled-components";
+import { Highlight } from "../sharedComponents/sharedComponents";
 
 export const ImpactWrapper = styled.section`
     background: #F4F4F4;
     padding: 120px 0;
     display: flex;
     flex-direction: column;
+    z-index: 0;
 `
 export const BoxWrapper = styled.div`
     display: flex;
+    position: relative;
     width: 94.44%;
     justify-content: space-between;
     margin: 60px auto 120px auto;
+
+    @media (max-width: 600px) {
+        flex-direction: column;
+    }
+`
+export const BoxContain = styled.div`
+    position: relative;
+    width: 48.53%;
+
+    @media (max-width: 600px) {
+        width: 75%;
+        margin: 0 auto 80px;
+    }
 `
 export const Box = styled.div`
+    position: relative;
+    z-index: 2;
     display: flex;
     flex-direction: column;
     justify-content: space-around;
-    width: 48.53%;
+    width: 100%;
+    height: 100%;
     align-items: center;
     background: #FFFFFF;
     box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.2);
     border-radius: 20px;
 `
-export const BoxTitle = styled.p`
+export const ImageWrap = styled.div`
+    position: absolute;
+    width: 36.52%;
+    top: -23.79%;
+    left: 31.67%;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    z-index: 0;
+    visibility: ${props => props.square};
+
+    @media (max-width: 1024px) {
+        top: -20%;
+    }
+
+    @media (max-width: 900px) {
+        top: -14%;
+    }
+
+    @media (max-width: 425px) {
+        top: -10%;
+    }
+`
+export const BoxImage = styled.img`
+    width: 31.12%;
+`
+export const Circles = styled.img`
+    position: absolute;
+    width: 37.73%;
```

Comment:
The circles should be higher at 1440 and 375. They look very close at 1024 and 768 but the design looks like they are lower.